designer/server/src/routes/forms/notification-email.js

```diff
@@ -0,0 +1,121 @@
+import { emailAddressSchema } from '@defra/forms-model'
+import { StatusCodes } from 'http-status-codes'
+import Joi from 'joi'
+
+import * as scopes from '~/src/common/constants/scopes.js'
+import { sessionNames } from '~/src/common/constants/session-names.js'
+import { buildErrorDetails } from '~/src/common/helpers/build-error-details.js'
+import * as forms from '~/src/lib/forms.js'
+import { notificationEmailViewModel } from '~/src/models/forms/notification-email.js'
+
+export const ROUTE_PATH_EDIT_NOTIFICATION_EMAIL =
+  '/library/{slug}/edit/notification-email'
+
+export const schema = Joi.object().keys({
+  notificationEmail: emailAddressSchema.required().messages({
+    'string.empty': 'Enter an email address submitted forms should be sent to',
+    'string.email':
+      'Enter an email address submitted forms should be sent to in the correct format, like name@gov.uk'
+  })
+})
+
+export default [
+  /**
+   * @satisfies {ServerRoute<{ Params: { slug: string } }>}
+   */
+  ({
+    method: 'GET',
+    path: ROUTE_PATH_EDIT_NOTIFICATION_EMAIL,
+    async handler(request, h) {
+      const { auth, params, yar } = request
+      const { slug } = params
+      const { token } = auth.credentials
+
+      const validation = yar
+        .flash(sessionNames.validationFailure.notificationEmail)
+        .at(0)
+
+      // Retrieve form by slug
+      const metadata = await forms.get(slug, token)
+
+      // Create the notification email view model
+      const model = notificationEmailViewModel(metadata, validation)
+
+      return h.view('forms/notification-email', model)
+    },
+    options: {
+      auth: {
+        mode: 'required',
+        access: {
+          entity: 'user',
+          scope: [`+${scopes.SCOPE_WRITE}`]
+        }
+      }
+    }
+  }),
+
+  /**
+   * @satisfies {ServerRoute<{ Params: { slug: string }, Payload: Pick<FormMetadataInput, 'notificationEmail'> }>}
+   */
+  ({
+    method: 'POST',
+    path: ROUTE_PATH_EDIT_NOTIFICATION_EMAIL,
+    async handler(request, h) {
+      const { auth, params, payload, yar } = request
+      const { slug } = params
+      const { notificationEmail } = payload
+      const { token } = auth.credentials
+
+      // Retrieve form by slug
+      const { id } = await forms.get(slug, token)
+
+      // Update the metadata with the notification email
+      await forms.updateMetadata(id, { notificationEmail }, token)
+
+      yar.flash(
+        sessionNames.successNotification,
+        'Email address for sending submitted forms has been added'
+      )
+
+      return h.redirect(`/library/${slug}`).code(StatusCodes.SEE_OTHER)
```

Comment:
Same here. If you'd like to follow 7b25501340abd0a587bcc3c826cda7ba2bb85f68, you can now use `formOverviewPath(slug)` here